File: capacitor-plugins/capacitor-native/ios/Sources/AonsokuNativePlugin/Data/Sync/SyncTier.swift
import Foundation

enum SyncTier: String, CaseIterable {
    case t1
    case t2
    case t3

    var freshWindowSeconds: TimeInterval {
        switch self {
        case .t1: return 5 * 60
        case .t2: return 30 * 60
        case .t3: return 2 * 60 * 60
        }
    }

    var freshWindowMs: Int {
        Int(freshWindowSeconds * 1000)
    }
}

File: capacitor-plugins/capacitor-native/android/src/main/java/github/realtvop/aonsoku/plugins/data/sync/SyncTier.kt
package github.realtvop.aonsoku.plugins.data.sync
enum class SyncTier(val freshWindowMs: Long) {
    T1(5 * 60 * 1000L), T2(30 * 60 * 1000L), T3(2 * 60 * 60 * 1000L)
}